.pipelines/templates/.builder-release-template-windows.yaml

```diff
@@ -115,6 +116,7 @@ steps:
         targetPath: 'release-notes.txt'
 
 # Default Gen 1: packer VHD=>VHD in classic SA
+# Credentials are without the BUILD prefix because we need AME account to copy to the classic SA
```

Comment:
Actually, we do not need any credential for copy since both src and dest have the sas token. I think that the credential is passed since it is needed for az login and it may be used. But for now, we do not use it.

Reply:
Will remove it.